### ISSUE TITLE
Optional JS for like/boost/reply buttons

### ISSUE THREAD

**Issue by BaptisteGelez** [Plume-org/Plume#591]
**Is your feature request related to a problem? Please describe.**

When you click one if these buttons on the article page, the page is realoaded and you are sent back at the top.

**Describe the solution you'd like**

Adding a little script that does the same request but in background, and updates the interface as soon as you clicked, without reloading the page and sending you back at the top.

**~~Describe alternatives you've considered~~**

**~~Additional context~~**

**Comment by fdb-hiroshima** [comment]
As an alternative which works without js : use an anchor to get them directly to the form instead of going back up

**Comment by BaptisteGelez** [comment]
Yeah, that would be nice to have too, but you would still see the page blinking because it reloads.

**Comment by AnaGelez** [comment]
@MasterOfTheTiger wants to work on this (I can't assign this issue to someone who is not in the Plume-org group :expressionless:)

**Comment by MasterOfTheTiger** [comment]
Oh. You cannot? That's odd. 

I will work on this, and hopefully get a fix in within the next couple of weeks.

**Comment by MasterOfTheTiger** [comment]
As you can see. No fix was provided. I am sadly unable to work on this. Please release this for someone else to do.
